lib/Transforms/Instrumentation/ShadowMemSeaDsa.cc

```diff
@@ -48,6 +48,15 @@ llvm::cl::opt<bool>
                      llvm::cl::desc("Use TypeBasedAA in the MemSSA optimizer"),
                      llvm::cl::init(true));
 
+namespace seahorn {
+bool InterProcMemFlag;
+}
+
+static llvm::cl::opt<bool,true>  InterProcMem("horn-inter-proc-mem",
```

Comment:
you do not need `1InterProcMemFlag`. You can use `InterProcMem` as a boolean directly.

Reply:
I see that the flag is needed because it is accessed elsewhere. Not ideal, but ok for now. Should add comment to that effect here where the variable is created.